prow/crier/reporters/gerrit/reporter.go

```diff
@@ -230,14 +230,17 @@ func (c *Client) ShouldReport(ctx context.Context, log *logrus.Entry, pj *v1.Pro
 	// current prowjob doesn't have this label or has an invalid value, this
 	// will be reflected as warning message in prow.
 	patchsetNumFromPJ := func(pj *v1.ProwJob) int {
+		log := log.WithFields(logrus.Fields{"label": kube.GerritPatchset, "job-name": pj.Name})
```

Comment:
Nit: "job-name" sounds like it is referring to `pj.Spec.Job`.
Can we keep this consistent with the field names we use elsewhere instead? https://github.com/kubernetes/test-infra/blob/43c5445b6733d639dadaa46d923109524cc29613/prow/pjutil/pjutil.go#L296-L299

Reply:
doen